src/dataServices/dynamoDbDataService.ts

```diff
@@ -149,12 +149,7 @@ export class DynamoDbDataService implements Persistence, BulkDataAccess {
     async updateResource(request: UpdateResourceRequest) {
         const { resource, resourceType, id } = request;
         const resourceCopy = { ...resource };
-        const getResponse = await this.readResource({ resourceType, id });
-        const currentVId: number = getResponse.resource.meta
-            ? parseInt(getResponse.resource.meta.versionId, 10) || 0
-            : 0;
-
-        resourceCopy.meta = generateMeta((currentVId + 1).toString());
+        await this.readResource({ resourceType, id });

```

Comment:
add a comment saying `// Will throw NotFoundError if resource can't be found`

Reply:
updated